City-of-Helsinki/drupal-hdbt
# UHF-1722 Arrow icon for hero

NOTICE: Do not merge before the UHF-1734-basic-page-changes has been merged to avoid the arrow from going over the menus on pages with those elements and such.

To test:

1. Switch to this branch `composer require drupal/hdbt:dev-UHF-1722-arrow-icon-for-hero`
2. Clear caches `make drush-cr`
3. Go to view any page or landing page that has hero. There should be arrow on the bottom-left corner of the hero. You need to check every hero-design separately since they differ quite a lot and this affects the arrow.
4. The koro-figures have also been modified in this PR so that they figure is now placed on the center instead of the top. Try the different koros the same time as you are testing the arrow icon. (/admin/tools/site-settings)
